extensions/multi-links/src/multi-links.ts

```diff
@@ -0,0 +1,18 @@
+import { getSelectedText, open, showHUD } from "@raycast/api";
+import extractUrls from "extract-urls";
+
+export default async function main() {
+  try {
+    const selectedText = await getSelectedText();
+    const urls = extractUrls(selectedText);
+    if (urls.length > 0) {
+      for (const url of urls) {
+        await open(url, "company.thebrowser.Browser");
```

Comment:
Why not open it in the default browser instead?

Reply:
Done!